testsuite/smoke-test/src/storage.rs

```diff
@@ -466,14 +466,16 @@ async fn do_transfer_or_reconfig(info: &mut AptosPublicInfo) -> Result<()> {
     const LOTS_MONEY: u64 = 100_000_000;
     let r = rand::random::<u64>() % 10;
     if r < 3 {
-        // reconfig
+        info!(
+            "{LINE} background task: triggering reconfig. Root account seq_num: {}. Ledger info: {:?}",
+            info.root_account().sequence_number(),
+            info.client().get_ledger_information().await.unwrap(),
+        );
         info.reconfig().await;
-    } else if r == 9 {
-        // drain backlog
-        let mut sender = info.create_and_fund_user_account(LOTS_MONEY).await?;
-        let receiver = info.create_and_fund_user_account(LOTS_MONEY).await?;
-        let pending_txn = info.transfer(&mut sender, &receiver, 1).await?;
-        info.client().wait_for_transaction(&pending_txn).await?;
```

Comment:
it's not draining backlog because this branch doesn't know the accounts used in the `else` branch. in fact, even in that branch the minting is waited on so we should not worry about mempool being too large.